src/Symfony/Component/Translation/LoggingTranslator.php

```diff
@@ -40,6 +46,7 @@ public function __construct($translator, LoggerInterface $logger)
 
         $this->translator = $translator;
         $this->logger = $logger;
+        $this->excludedDomains = !empty($excludedDomains) ? '{('.implode('|', $excludedDomains).')}i' : false;
```

Comment:
are you expecting an array of domains or an array of regexes ? The name suggests the former but the code implements the later.

and this should use a non-capturing group